### DISCUSSION TITLE
How do I merge specified cells

### DISCUSSION THREAD

**Discussion by StringAech** [rstudio/gt#1150] [Q&A]
hey guys!
i want merge some specified cells just like **colspan="2"** in HTML, but there only can merge the all columns by use **col_merge()**.
what should i do ?
for example:
i want this table:
<table>
    <thead>
        <tr>
            <th>Layer 1</th>
            <th>Layer 2</th>
            <th>Layer 3</th>
        </tr>
            <th>Layer 4</th>
            <th>Layer 5</th>
            <th>Layer 6</th>
</table>
to change like this 
<table>
    <thead>
        <tr>
            <th>Layer 1</th>
            <th>Layer 2</th>
            <th>Layer 3</th>
        </tr>
            <th colspan=2>Layer 4</th>
            <th>Layer 5</th>
</table>
or
<table>
    <thead>
        <tr>
            <th rowspan=2>Layer 1</th>
            <th>Layer 2</th>
            <th>Layer 3</th>
        </tr>
            <th>Layer 5</th>
            <th>Layer 6</th>
</table>

**Comment by ixodid198** [comment]
I too have this problem. I posted on Stackoverflow and @MarBlo posted a possible [solution](https://stackoverflow.com/questions/74846525/merge-two-cells-together-in-gt-in-just-one-row). However, I like simplicity so I went with `flextable`.

**Reply by StringAech** [reply]
Thank you very much for your answer!
` flextable `can help me a lot.
